pkg/process/checks/net.go

```diff
@@ -126,12 +131,82 @@ func (c *ConnectionsCheck) Realtime() bool { return false }
 // ShouldSaveLastRun indicates if the output from the last run should be saved for use in flares
 func (c *ConnectionsCheck) ShouldSaveLastRun() bool { return false }
 
+func (c *ConnectionsCheck) handleBatch(batch *model.Connections, start time.Time, groupID int32, isFirst bool) model.MessageBody {
```

Comment:
does it make sense to split handleBatch from its caller?

Reply:
changed it to be included in the caller area